src/EventsHeader/EventsHeader.jsx

```diff
@@ -0,0 +1,262 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import i18n from '../i18n';
+import sanitizeHtml from 'sanitize-html';
+import { Typography, Button, withStyles, Hidden } from '@material-ui/core';
+import Container from '../Container/Container';
+import ResponsiveImage from '../ResponsiveImage/ResponsiveImage';
+import { Link, Element } from 'react-scroll';
+import { translate } from 'react-i18next';
+import EventsHeaderButton from './EventsHeaderButton';
+import css from './EventsHeader.module.css';
+
+// https://material.io/tools/icons/?style=baseline
+import {
+	Edit as EditIcon,
+	Event as EventIcon,
+	Movie as MovieIcon,
+	Portrait as PortraitIcon,
+} from '@material-ui/icons';
+import { purple } from '@material-ui/core/colors';
+
+const styles = theme => ({
+	button: {
+		color: theme.palette.getContrastText(purple[500]),
+		backgroundColor: purple[500],
+		'&:hover': {
+			color: 'white',
+			backgroundColor: purple[700],
+		},
+	},
+});
+
+const navButtons = [
```

Comment:
Would it be neat to create a constants file for these kind of things? Like a constants file specifically for the events module on our website?

Reply:
Then you can add it as a prop to the Header component and render it there.

Reply:
Why not keep it as a constants.jsx next to EventsHeader.jsx and import it there? This is the only place where it is used at the moment. Maybe later, when there are more constants for Events, we can move to this approach.